Platforms/Apple/src/AppleFileSystem.cpp

```diff
@@ -125,14 +127,28 @@ bool AppleFileSystem::FileExists(const Char* strFilePath)
 
 bool AppleFileSystem::PathExists(const Char* strPath)
 {
-    UNSUPPORTED("Not implemented");
-    return false;
+    std::string path(strPath);
+    CorrectSlashes(path, AppleFileSystem::GetSlashSymbol());
+
+    auto res = access(path.c_str(), R_OK);
+    return res == 0;
 }
 
 bool AppleFileSystem::CreateDirectory(const Char* strPath)
 {
-    UNSUPPORTED("Not implemented");
-    return false;
+    std::string path(strPath);
+    CorrectSlashes(path, AppleFileSystem::GetSlashSymbol());
+
+    auto res = mkdir(path.c_str(), 0777);
```

Comment:
Please use named constants instead of 777.
This should be S_IRWXU | S_IRWXG | S_IRWXO I think - double check